Sources/Orbit/Foundation/Elevations/Elevation.swift

```diff
@@ -0,0 +1,176 @@
+import SwiftUI
+
+/// Use elevation to bring content closer to users.
+///
+/// Elevation levels with higher numbers are usually visually closer to the user.
+public enum Elevation {
+    case level1
+    case level2
+    case level3
+    case level4
+}
+
+public extension View {
+
+    /// Elevates the view to make it more prominent.
+    func elevation(_ level: Elevation) -> some View {
+        elevation(level, prerender: true)
+    }
+}
+
+extension View {
+
+    func elevation(_ level: Elevation, prerender: Bool) -> some View {
+        modifier(ElevationModifier(level: level, isPrerendered: prerender))
+    }
+}
+
+struct ElevationModifier: ViewModifier {
+
+    @Environment(\.isElevationEnabled) var isElevationEnabled
+    let level: Elevation
+    let isPrerendered: Bool
+    let shadowColor = Color(red: 79 / 255, green: 94 / 255, blue: 113 / 255)
+
+    func body(content: Content) -> some View {
+        if isElevationEnabled {
+            if isPrerendered {
+                content
+                    .background(prerenderedShadow(content: content))
+            } else {
+                content
+                    .background(dynamicShadow(content: content))
+            }
+        } else {
+            content
+        }
+    }
+
+    @ViewBuilder func prerenderedShadow(content: Content) -> some View {
+        switch level {
+            case .level1:
+                content
+                    .shadow(color: shadowColor.opacity(0.28), radius: 2.6, y: 1.7)
+                    .padding(.small)
+                    .drawingGroup(colorMode: .extendedLinear)
+            case .level2:
+                content
+                    .shadow(color: shadowColor.opacity(0.18), radius: 4, y: 4)
+                    .shadow(color: shadowColor.opacity(0.12), radius: 8, y: 8)
+                    .padding(.medium)
+                    .drawingGroup(colorMode: .extendedLinear)
+            case .level3:
+                content
+                    .shadow(color: shadowColor.opacity(0.12), radius: 4, y: 4)
+                    .shadow(color: shadowColor.opacity(0.11), radius: 8, y: 8)
+                    .shadow(color: shadowColor.opacity(0.10), radius: 16, y: 16)
+                    .padding(.xLarge)
+                    .drawingGroup(colorMode: .extendedLinear)
+            case .level4:
+                content
+                    .shadow(color: shadowColor.opacity(0.12), radius: 4, y: 4)
+                    .shadow(color: shadowColor.opacity(0.11), radius: 8, y: 8)
+                    .shadow(color: shadowColor.opacity(0.10), radius: 16, y: 16)
+                    .shadow(color: shadowColor.opacity(0.09), radius: 32, y: 32)
+                    .padding(.xxxLarge)
+                    .drawingGroup(colorMode: .extendedLinear)
+        }
+    }
+
+    @ViewBuilder func dynamicShadow(content: Content) -> some View {
```

Comment:
Suggestion for dynamic version:

```

    @ViewBuilder func dynamicShadow(content: Content) -> some View {
        switch level {
            case .level1:
                content
                    .shadow(color: shadowColor.opacity(0.12), radius: 1, y: 1)
                    .shadow(color: shadowColor.opacity(0.11), radius: 2, y: 2)
                    .shadow(color: shadowColor.opacity(0.10), radius: 4, y: 4)
            case .level2:
                content
                    .shadow(color: shadowColor.opacity(0.12), radius: 2, y: 2)
                    .shadow(color: shadowColor.opacity(0.11), radius: 4, y: 4)
                    .shadow(color: shadowColor.opacity(0.10), radius: 8, y: 8)
//                    .shadow(color: shadowColor.opacity(0.09), radius: 16, y: 16)
            case .level3:
                content
                    .shadow(color: shadowColor.opacity(0.12), radius: 2, y: 2)
                    .shadow(color: shadowColor.opacity(0.11), radius: 4, y: 4)
                    .shadow(color: shadowColor.opacity(0.10), radius: 8, y: 8)
                    .shadow(color: shadowColor.opacity(0.06), radius: 16, y: 16)
//                    .shadow(color: shadowColor.opacity(0.08), radius: 32, y: 32)
            case .level4:
                content
                    .shadow(color: shadowColor.opacity(0.12), radius: 2, y: 2)
                    .shadow(color: shadowColor.opacity(0.11), radius: 4, y: 4)
                    .shadow(color: shadowColor.opacity(0.10), radius: 8, y: 8)
                    .shadow(color: shadowColor.opacity(0.09), radius: 16, y: 16)
                    .shadow(color: shadowColor.opacity(0.06), radius: 32, y: 32)
//                    .shadow(color: shadowColor.opacity(0.07), radius: 64, y: 64)
        }
    }
```

<img width="1066" alt="image" src="https://user-images.githubusercontent.com/35844477/175267735-01ddf741-e9da-4f89-8b4b-9d5ecdd4c8bc.png">